index.html

```diff
@@ -120,26 +128,46 @@ <h2>Контакты</h2>
       <p>наб. Реки Карповки,
       <br>5 лит П, Санкт-Петербург</p>
```

Comment:
Тег `<br>` лучше ставить на строке, с которой производится перевод строки. Иначе обычный перевод строки в и пробелы перед следующей строкой (перед `<br>`) конвертируются браузером в один обычный пробел.

![image](https://user-images.githubusercontent.com/3382798/120061431-e6b84d00-c065-11eb-96bd-e7030451ccf0.png)

Это может показаться незначительным, но если поднадобится выравнивание (выключка) не по левому краю, то это пробел будет мешаться.
```suggestion
      <p>наб. Реки Карповки,<br>
      5 лит П, Санкт-Петербург</p>
```